cmds/verify_strings.go

```diff
@@ -214,6 +214,10 @@ func (vs *verifyStrings) verify(inputFilename string, targetFilename string) err
 	if err != nil {
 		return fmt.Errorf("File has duplicated key: %s\n%s", inputFilename, err)
 	}
+	return vs.Verify(inputMap, targetFilename)
+}
+
+func (vs *verifyStrings) Verify(inputMap map[string]I18nStringInfo, targetFilename string) error {
```

Comment:
What about this targetFilename? Does it mean that verify will continue operating with files? I see some generateFile functions, so is it desirable to have a Verify function creating files?

Reply:
Ah yeah let me explain a bit! There are two files involved here:
1. The input file, containing all the translations that we want to check against
2. The translation strings file, containing all the existing strings for a given locale

2 is called "targetFilename" in this function, it's read from the filesystem. 1 can either be read from the filesystem (by using the existing Run() method), or passed in as a map (by using this Verify() method). We still generate a .diff file in both cases, unless the "dry run" flag is set.

Making this whole module work with in-memory stuff instead of writing temporary files all over the place is a longer-term goal of mine, this is a step in that direction but certainly not the only step involved 😅